### ISSUE TITLE
[Question] Inquiry About the Release of Training Scripts

### ISSUE THREAD

**Issue by heyy-sun** [JarrentWu1031/AnimateAnyMesh#20]
Thank you for your incredible work on AnimateAnyMesh! I've been testing the inference code, and it runs perfectly. I'm very interested in taking a deeper dive into the training process itself. Could you let me know if you plan on open-sourcing the training code as well? I'd love to learn from it and try it out on my own data.

**Comment by Prady1272** [comment]
Hi, is there a plan to release the test split?

**Comment by JarrentWu1031** [comment]
The training code is released!

**Comment by JarrentWu1031** [comment]
> Hi, is there a plan to release the test split?

Hi! Sorry for being busy recently! The val/test set will be released soon!
